R/hush.R

```diff
@@ -1,6 +1,6 @@
 # Set an option to control whether messages and warning are suppressed.
 # Defaults to verbose messaging.
-options("matilda.verbose" = FALSE)
+options("matilda.verbose" = TRUE)
```

Comment:
Huh, I've never seen quotes used with `options`. If it's working, cool, but normally I feel like most R coders would write
```r
options(matilda.verbose = TRUE)
```